src/app/features/test-records/amend/views/test-record/test-record.component.spec.ts

```diff
@@ -163,15 +163,15 @@ describe('TestRecordComponent', () => {
       store.overrideSelector(testResultInEdit, testRecord);
       store.overrideSelector(sectionTemplates, Object.values(masterTpl.psv['testTypesGroup1']!));
 
-      tick();
+      tick(1000);
```

Comment:
Is this needed?

Reply:
This needs to be in otherwise the test fails